llama_index/indices/postprocessor/sbert_rerank.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Optional
+from llama_index.indices.postprocessor.types import BaseNodePostprocessor
+from llama_index.indices.query.schema import QueryBundle
+from llama_index.schema import MetadataMode, NodeWithScore
+
+DEFAULT_SENTENCE_TRANSFORMER_MAX_LENGTH = 512
+
+
+class SentenceTransformerRerank(BaseNodePostprocessor):
+    def __init__(
+        self,
+        top_n: int = 2,
+        model: str = "cross-encoder/ms-marco-TinyBERT-L-2-v2",
```

Comment:
Any reason why this is the default, but the docs use a different model?

Reply:
I explained in the docs. I would prefer if the user looks up the sentence-transformer page to make their choice.

Reply:
Gotcha, that's what I wanted to know :)